extension/src/popup/views/ManageAssetsLists/index.tsx

```diff
@@ -63,6 +64,18 @@ export const ManageAssetsLists = () => {
     fetchLists();
   }, [selectedNetwork, assetsLists]);
 
+  useEffect(() => {
+    if (networkDetails.network === NETWORKS.TESTNET) {
```

Comment:
Is this check already taken care of by the next `setSelectedNetwork`?

Reply:
Oh whoops - my bad, meant to remove this!